### PR TITLE
Add performance tests 

### PR DESCRIPTION
**Pull request by nytian**
Add Durable Functions Performance tests to the extension repository for timely update.

### CODE REVIEW

**Review by davidmrdavid — COMMENTED** [review]
Thanks for opening these PR. Just left some minor feedback

File: test/PerfTests/DFPerfTests/DFPerfScenarios/HelloWorld.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            var input = ctx.GetInput<dynamic>();
+            return $"{input.Hello}, {input.Name}";
+        }
+
+        [FunctionName("FailedActivity")]
+        public string FailedActivity(
+    [ActivityTrigger] IDurableActivityContext ctx)
+        {
+            return null;
+        }
```
Comment (davidmrdavid) [inline test/PerfTests/DFPerfTests/DFPerfScenarios/HelloWorld.cs:29]: Can we format this file to make sure the `ActivityTrigger` in line 26 is indented similarly to the `OrchestrationTrigger` in line 18? I believe VS should do this for you

File: test/PerfTests/DFPerfTests/host.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          "controlQueueBufferThreshold": 1000,
+          "storageProvider": {
+            "partitionCount": 6,
+          }
+        }},
+    "logging": {
+        "applicationInsights": {
+            "samplingSettings": {
+                "isEnabled": true,
+                "excludedTypes": "Request"
```
Comment (davidmrdavid) [inline test/PerfTests/DFPerfTests/host.json:22]: the `logging` section in line 18 seems under indented. Can we format this file? I believe the editor can do this for you as well

File: test/PerfTests/DFPerfTests/Counter.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            [DurableClient] IDurableClient starter,
+            ILogger log)
+		{
+			Input input = await req.Content.ReadAsAsync<Input>();
+			if (input == null || input.EventCount <= 0)
+			{
+				return req.CreateErrorResponse(HttpStatusCode.BadRequest, "A positive integer was expected for EventCount.");
+			}
+
+            var entityId = new EntityId("Counter", Guid.NewGuid().ToString("N"));
```
Comment (davidmrdavid) [inline test/PerfTests/DFPerfTests/Counter.cs:27]: The indentation here seems a bit off. Can we format this file as well?

**Review by davidmrdavid — COMMENTED** [review]
Almost there. There are still some files that need formatting. I've called them out

**Comment by nytian** [comment]
> Almost there. There are still some files that need formatting. I've called them out

Thanks for pointing them out.

**Review by davidmrdavid — APPROVED** [review]

**Review by bachuv — APPROVED** [review]
Thanks for adding the performance tests!

**Pull request merged**